src/lineup/internal/panel/PanelDownloadButton.ts

```diff
@@ -0,0 +1,60 @@
+import {LocalDataProvider} from 'lineupjs';
+import {exportLogic} from '../export';
+import {IPanelButton} from './PanelButton';
+import i18n from 'phovea_core/src/i18n';
+
+
+/**
+ * A button dropdown to download selected/all rows of the ranking
+ */
+export default class PanelDownloadButton implements IPanelButton {
+  readonly node: HTMLElement;
+
+  constructor(parent: HTMLElement, private provider: LocalDataProvider, isTopMode:boolean) {
+    this.node = parent.ownerDocument.createElement('div');
+    this.node.classList.add('btn-group', 'download-data-dropdown');
+    this.node.innerHTML = `
```

Comment:
why aren't you inserting the elements via `someHTMLNode.insertAdjacentHTML('afterbegin', '...')` [like you're doing here](https://github.com/datavisyn/tdp_core/blob/ochampari/10_tourdino-lineup-panel-button/src/lineup/internal/panel/PanelButton.ts#L54) or vice versa?

Reply:
No special reason behind it. This can be unified.

Reply:
No special reason, it can be unified.